todo/admin.py

```diff
@@ -1,13 +1,41 @@
 from django.contrib import admin
 
 from todo.models import Attachment, Comment, Task, TaskList
-
+import csv
+import datetime
+from django.http import HttpResponse
+from django.urls import reverse
+from django.utils.safestring import mark_safe
+
+
+def export_to_csv(modeladmin, request, queryset):
+    opts = modeladmin.model._meta
+    content_disposition = f'attachment; filename={opts.verbose_name}.csv'
+    response = HttpResponse(content_type='text/csv')
+    response['Content-Disposition'] = content_disposition
+    writer = csv.writer(response)
+    fields = [field for field in opts.get_fields() if not field.many_to_many\
```

Comment:
Prefer not to use those old-school backslashes for newlines, but I'll make a little tweak after it's merged.